README.md

```diff
@@ -2,21 +2,17 @@
 
 [![Zulip](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://hail.zulipchat.com?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
-[Hail](https://hail.is) is an open-source, general-purpose, Python-based data analysis tool with additional data types and methods for working with genomic data. Hail is used throughout academia and industry as the analytical engine for major studies, projects, and services, including the Genome Aggregation Database ([gnomad.broadinstitute.org](http://gnomad.broadinstitute.org)) and Neale lab mega-GWAS ([nealelab.is/uk-biobank](https://nealelab.is/uk-biobank)).
+[Hail](https://hail.is) is an open-source, general-purpose, Python-based data analysis tool with additional data types and methods for working with genomic data.
 
-Unlike the Python and R scientific computing stacks, Hail:
+Hail is built to scale, and has first-class support for multi-dimensional structured data, like genomic data as in the genome-wide association study (GWAS) [tutorial](https://hail.is/docs/0.2/tutorials/01-genome-wide-association-study.html)
```

Comment:
Hail is built to scale and has first-class support for multi-dimensional structured data, like the genomic data of a genome-wide association study (GWAS).